apps/fyle/signals.py

```
@@ -0,0 +1,25 @@
+"""
+Fyle Signals
+"""
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from fyle_accounting_mappings.models import DestinationAttribute, EmployeeMapping
```

Ruff (0.8.2)

`fyle_accounting_mappings.models.DestinationAttribute` imported but unused

Remove unused import

(F401)